<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QwtPlot" name="plotter"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Num. Reads</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="num_reads"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Trigger</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="trigger"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Range</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="range"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Num. Samples</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="num_samples"/>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="rerun_btn">
         <property name="text">
          <string>Rerun</string>
         </property>
        </widget>
        <widget class="QPushButton" name="close_btn">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
